ggshield/cmd/iac/scan/iac_scan_common_options.py

```diff
@@ -26,7 +30,10 @@
 _exit_zero_option = click.option(
     "--exit-zero",
     is_flag=True,
-    help="Always return 0 (non-error) status code.",
+    default=None,
```

Comment:
I think that the default value should be False rather than None.

Reply:
I agree. However, when setting the default to `False`, the option flag isn't working anymore.
In other parts of the code (in secrets), flags always have a default value of None.
It seems some functions do not work with a non-None default.

I suggest we keep the None default for now.